### ISSUE TITLE
1 list with many tags instead of many lists

### ISSUE THREAD

**Issue by nickjj** [freeCodeCamp/mail-for-good#197]
This is probably a huge change in terms of coding but for the last couple of years other platforms have shifted towards using tags instead of individual lists to manage subscribers.

I think this is a really good idea because it puts the emphasis on the subscriber instead of the list, which in turn makes it WAY easier to get useful information from your subscribers and even manage and market to your subscribers (no need to keep making more lists).

Think about it like this.

Imagine if you have a subscriber with many tags.

Suddenly you can look at that subscriber at a glance and see a list of tags, for example: "purchased foobar", "purchased wizbang", "downloaded hello guide". You can see everything about it at a glance. When they performed the action, how many times they clicked links, etc..

Now you can determine "oh wow this person bought 2 things and is interested in other free content too". This allows you to send highly segmented emails out to "subscribers who bought these 2 products" or whatever works for your company.

Sure, you can do this with lists but you end up with a lot of duplicate data because that subscriber exists on each individual list. It becomes a nightmare to try and stitch together reports across multiple lists. You would also need to configure your campaigns to work across multiple lists and remove duplicate emails (not sure if your product even does this?).

Basically the major win for having it set up with tags is a single subscriber has many tags, and you can get very excellent reporting / activity feeds / segmentation at a glance.

**Comment by karuppiah7890** [comment]
It's really a nice idea. Also, I think we can still have many lists feature even after adding tags, just in case people have non-overlapping subscribers (like for different non-related sites/products)

**Comment by nickjj** [comment]
Right, that makes sense. This way 1 installation can manage multiple projects.

Here's documentation from other platforms that describe tags in general (for inspiration):

http://kb.getdrip.com/general/overview-of-tags-events-and-custom-fields/
http://help.convertkit.com/article/12-convertkit-subscribers#f4
